tokio/src/util/sharded_list.rs

```diff
@@ -0,0 +1,160 @@
+#![cfg_attr(not(feature = "full"), allow(dead_code))]
+
+use std::{ptr::NonNull, sync::atomic::Ordering};
```

Comment:
We only use `{}` for things in the same module.
```suggestion
use std::ptr::NonNull;
use std::sync::atomic::Ordering;
```